crates/iota-node/src/lib.rs

```diff
@@ -249,6 +254,11 @@ pub struct IotaNode {
     // Channel to allow signaling upstream to shutdown iota-node
     shutdown_channel_tx: broadcast::Sender<Option<RunWithRange>>,
 
+    /// Broadcast channels for gRPC checkpoint streaming
+    grpc_checkpoint_summary_tx:
+        Option<tokio::sync::broadcast::Sender<Arc<CertifiedCheckpointSummary>>>,
```

Comment:
We should import `tokio::sync::broadcast` and use that everywhere.

Reply:
Agreed. Fixed at [b1fb10d](https://github.com/iotaledger/iota/pull/7511/commits/b1fb10d7a7ee183e4e92a5f2673d2bef6a7b788d).